src/Domain/Value/Resolver/RelationCollection.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of Storyblok-Api.
+ *
+ * (c) SensioLabs Deutschland <info@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Storyblok\Api\Domain\Value\Resolver;
+
+/**
+ * @author Silas Joisten <silasjoisten@proton.me>
+ *
+ * @implements \IteratorAggregate<int, Relation>
+ */
+final class RelationCollection implements \Countable, \IteratorAggregate
```

Comment:
```suggestion
final class Relations implements \Countable, \IteratorAggregate
```

just an idea

Reply:
Yea we have a lot of collections in there and they are all suffixed with Collection so to keep it consistent i prefer to keep the Suffix